### PR TITLE
don't overwrite LD_LIBRARY_PATH when compiling dependencies

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: install/funcs.py [hunk 1]
```diff
@@ -476,7 +476,7 @@ def path(x):
 
         environ = {
             'PYTHONHOME': prefix,
-            'LD_LIBRARY_PATH': '%s/lib' % prefix,
+            'LD_LIBRARY_PATH': '%s/lib:$LD_LIBRARY_PATH' % prefix,
```
Comment (delarosatrevin) [inline install/funcs.py]: it should be something like os.environ.get('LD_LIBRARY_PATH')...we need to test it.

**Comment by pescobar** [comment]
I have done the modification suggested by @delarosatrevin 

I tested the change in my system and works for me. I could compile scipion without issues

**Comment by delarosatrevin** [comment]
Thanks @pescobar, all the other Python modules compiled without issues? It was CentOS 6.5, right?

**Comment by pescobar** [comment]
yes, everything built correctly. I could also install all the dependencies doing 

```
./scipion  install -j 8  bsoft-1.8.8 bsoft-1.9.0 chimera cryoem ctffind ctffind4 dogpicker eman2.11 eman2.12 frealign gEMpicker_v1.1 motioncorr nma relion-1.3 relion-1.4 relion-1.4_float resmap simple spider summovie unblur
```

The machine were I did the tests was centos 6.5 . The compiler I am using and other dependencies like CMake or GSL lib are setup using `LD_LIBRARY_PATH`  (and other env vars) defined using environment modules.

**Comment by azazellochg** [comment]
After this change, I also can compile scipion 1.0 (cloned from git) from sources on Ubuntu 14.04.4 with gcc 4.8.4 and cuda 5.5

**Comment by delarosatrevin** [comment]
Thanks to both of you guys. @pescobar , one last thing, could you do the Pull Request against v1.0 branch? We are trying to follow the gitflow, where master only contains release states. So this kind of bug-fixes are going to v1.0 branch that will be merged to master and will become release v1.0.1.

**Comment by pescobar** [comment]
I have sent the PR agains v1.0 branch here https://github.com/I2PC/scipion/pull/364

closing this one

**Comment by delarosatrevin** [comment]
Thanks!

**Pull request closed without merging**